From c54dfea25897b3b182c152cfdf3edbf479e84e40 Mon Sep 17 00:00:00 2001
Message-ID: <c54dfea25897b3b182c152cfdf3edbf479e84e40.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 01:36:33 +0100
Subject: [PATCH 19/19] gnu: go-github-com-alecthomas-assert: Move to (gnu
 packages golang-check).

* gnu/packages/golang-check.scm (go-github-com-alecthomas-assert): Move
from here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 35 +++++++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 31 -------------------------------
 2 files changed, 35 insertions(+), 31 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 8fc9d2fbc0..42e501a6e3 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -5,6 +5,7 @@
 ;;; Copyright © 2019 Leo Famulari <leo@famulari.name>
 ;;; Copyright © 2020 Jakub Kądziołka <kuba@kadziolka.net>
 ;;; Copyright © 2020 Joseph LaFreniere <joseph@lafreniere.xyz>
+;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2021 Guillaume Le Vaillant <glv@posteo.net>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
@@ -82,6 +83,40 @@ (define-public go-honnef-co-go-tools
 simplifications, and enforces style rules.")
     (license license:expat)))
 
+(define-public go-github-com-alecthomas-assert
+  (let ((commit "405dbfeb8e38effee6e723317226e93fff912d06")
+        (revision "1"))
+    (package
+      (name "go-github-com-alecthomas-assert")
+      (version (git-version "0.0.1" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/alecthomas/assert")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "1l567pi17k593nrd1qlbmiq8z9jy3qs60px2a16fdpzjsizwqx8l"))))
+      (build-system go-build-system)
+      (arguments
+       `(#:import-path "github.com/alecthomas/assert"))
+      (native-inputs
+       (list go-github-com-alecthomas-colour
+             go-github-com-mattn-go-isatty
+             go-github-com-alecthomas-repr
+             go-github-com-sergi-go-diff))
+      (home-page "https://github.com/alecthomas/assert/")
+      (synopsis "Go assertion library")
+      (description "Assertion library that:
+@itemize
+@item makes spotting differences in equality much easier
+@item uses repr and diffmatchpatch to display structural differences in colour
+@item aborts tests on first assertion failure
+@end itemize\n")
+      (license license:expat))))
+
 (define-public go-github-com-cheekybits-is
   (let ((commit "68e9c0620927fb5427fda3708222d0edee89eae9")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index aedf509c97..b32e379e47 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9219,37 +9219,6 @@ (define-public go-github-com-sergi-go-diff
 @end itemize\n")
     (license license:expat)))
 
-(define-public go-github-com-alecthomas-assert
-  (let ((commit "405dbfeb8e38effee6e723317226e93fff912d06")
-        (revision "1"))
-    (package
-      (name "go-github-com-alecthomas-assert")
-      (version (git-version "0.0.1" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/alecthomas/assert")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "1l567pi17k593nrd1qlbmiq8z9jy3qs60px2a16fdpzjsizwqx8l"))))
-      (build-system go-build-system)
-      (arguments
-       `(#:import-path "github.com/alecthomas/assert"))
-      (native-inputs
-       (list go-github-com-mattn-go-isatty go-github-com-alecthomas-colour
-             go-github-com-alecthomas-repr go-github-com-sergi-go-diff))
-      (home-page "https://github.com/alecthomas/assert/")
-      (synopsis "Go assertion library")
-      (description "Assertion library that:
-@itemize
-@item makes spotting differences in equality much easier
-@item uses repr and diffmatchpatch to display structural differences in colour
-@item aborts tests on first assertion failure
-@end itemize\n")
-      (license license:expat))))
-
 (define-public go-github-com-alecthomas-chroma
   (package
     (name "go-github-com-alecthomas-chroma")
-- 
2.41.0

